articles/ai-studio/how-to/fine-tune-serverless.md

```diff
@@ -154,6 +154,29 @@ The supported file type is JSON Lines. Files are uploaded to the default datasto
 
 Once your model is fine-tuned, you can deploy it and use it in your own application, in the playground, or in prompt flow. For more information on how to use deployed models, see [How to use Mistral premium chat models](./deploy-models-mistral.md).
 
+---
+## Supported Enterprise Scenarios for Finetuning
+
+Several enterprise scenarios are supported for MaaS finetuning. The table below outlines the supported configurations for user storage networking and authentication to ensure smooth operation within enterprise scenarios:
+
```

Comment:
See if it makes sense to link existing managed VNET setup documentation somewhere in this.